helm/cluster-vsphere/templates/helmreleases/kube-vip-cloud-provider-helmrelease.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: {{ include "resource.default.name" $ }}-kube-vip-cloud-provider
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    cluster.giantswarm.io/description: "{{ .Values.global.metadata.description }}"
+  labels:
+    cluster-apps-operator.giantswarm.io/watching: ""
+    giantswarm.io/service-priority: "{{ .Values.global.metadata.servicePriority }}"
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  suspend: {{ include "isIpamSvcLoadBalancerEnabled" $ }} # It is unsuspended by the post-install/post-upgrade hook.
```

Comment:
this helmrelease needs to be suspended until the service IPs have been claimed on the MC